package.json

```diff
@@ -89,32 +89,32 @@
     "json2yaml": "^1.1.0",
     "jsonwebtoken": "^8.5.1",
     "jwk-to-pem": "^2.0.5",
-    "lint-staged": "^12.3.5",
+    "lint-staged": "^12.4.0",
     "lodash": "^4.17.21",
     "logform": "^2.4.0",
-    "luxon": "^2.3.1",
+    "luxon": "^2.3.2",
     "moize": "^6.1.0",
     "nodemon": "^2.0.15",
     "npm-force-resolutions": "0.0.10",
     "npm-run-all": "^4.1.5",
     "object-sizeof": "^1.6.3",
-    "openapi-types": "^10.0.0",
+    "openapi-types": "^11.0.0",
     "pa11y-ci": "^3.0.1",
     "parse-full-name": "^1.2.6",
     "pdf-lib": "^1.17.1",
-    "pdfjs-dist": "~2.12.313",
+    "pdfjs-dist": "2.12.313",
```

Comment:
We want to pin the version to `2.12.313` exactly?

Reply:
yeah, this was documented in the `docs/team-process.md` section of the docs.  if we upgrade pdfjs-dist, the smoketests start failing because the library seems to have a bug in it.  I'm just locking it down for now since the next person to do a patch upgrade will run into the same issue